api/storage_service.cc

```diff
@@ -33,6 +33,8 @@
 #include "service/storage_service.hh"
 #include "service/load_meter.hh"
 #include "db/commitlog/commitlog.hh"
```

Comment:
Why bundle CALL and describe_ring? They are not strongly related.

Reply:
explained below